scripts/binary/binary-sources.js

```diff
@@ -82,11 +82,31 @@ const validateCloudApiFile = async (cloudApiFilePath) => {
   }
 }
 
+const getProtocolFileSource = async (protocolFilePath) => {
+  const fileContents = await fs.readFile(protocolFilePath, 'utf8')
+
+  if (!fileContents.includes('process.env.CYPRESS_LOCAL_PROTOCOL_PATH')) {
+    throw new Error(`Expected to find CYPRESS_ENV_PROTOCOL in protocol file`)
```

Comment:
is this error correct? Or does `CYPRESS_ENV_PROTOCOL` need to be set in order for a developer/user to leverage local protocol path?

Reply:
Doh.
```suggestion
    throw new Error(`Expected to find CYPRESS_LOCAL_PROTOCOL_PATH in protocol file`)
```